examples/platform/silabs/efr32/rs911x/rsi_wlan_config.h

```diff
@@ -57,15 +57,24 @@
 #ifdef RSI_M4_INTERFACE
 #define RSI_EXT_CUSTOM_FEATURE_BIT_MAP (EXT_FEAT_256K_MODE | EXT_FEAT_IEEE_80211W)
 #else
+#ifdef CHIP_9117
```

Comment:
This list of #if and #endif add of lot of confusion could you add some spacing in between and do a quick reformat ?

Reply:
Actually restyler is reverting it back to without spaces format. I have removed the M4 from there since it is never used in NCP version